CHANGELOG.md

```diff
@@ -24,6 +24,7 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 
 - Replace grommet Heading (#2410)
 - Replace all the loaders by Cunningham Loader (#2436)
+- Replace grommet Checkbox and Pagination (#2437)
```

Comment:
Be careful here ! :slightly_smiling_face:

Reply:
[4409657](https://github.com/openfun/marsha/pull/2437/commits/4409657850420e36546babc222f21535bab84fa0)